Gemfile

```diff
@@ -27,6 +27,9 @@ group :development, :test do
   gem 'rspec-rails' , '~> 3.3.0'
 end
 
-group :development do
+group :test do
+gem "shoulda-matchers", "< 3.0.0",
```

Comment:
@sairam49 `gem "shoulda-matchers", "< 3.0.0", require: false` this should be in one line. It basically means install this gem but don't load it during the application initialization. If in case you need to use the module/gem functionality at that time you'll have to require it.